### PR TITLE
EUI-4719 Replace hard coded backend url

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4719

### Change description ###
Global search - Replace hardcoded backend URL

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { HttpClientModule } from '@angular/common/http';
```
Comment (Daniel-Lam) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts:1]: @johnbenjamin-hmcts - this should be using `HttpClientTestingModule` for testing. I also made the same mistake and @paddybasi advised to use the above. I have committed the fix to the `feature/global-search` branch; please pull the latest.

File: src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts [hunk 2]
```diff
@@ -58,6 +57,7 @@ describe('HmctsGlobalHeaderComponent', () => {
       declarations: [ HmctsGlobalHeaderComponent ],
       schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
       imports: [
+        HttpClientModule,
```
Comment (Daniel-Lam) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts:60]: @johnbenjamin-hmcts - as above, please change this to `HttpClientTestingModule`.

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just two things to look at.

File: src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts]: Thank you @Daniel-Lam - I have now pulled the latest from the base branch and updated the code.

**Review by Daniel-Lam — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approve

**Pull request merged**